.mergify.yml

```diff
@@ -27,3 +28,18 @@ pull_request_rules:
       backport:
         branches:
           - release-v1.2.0
+  # automerge backports if CI successfully ran
+  - name: automerge backport release-v1.2.0
+    conditions:
+      - author=mergify[bot]
+      - base=release-v1.2.0
+      - label!=DNM
+      - "#changes-requested-reviews-by=0"
```

Comment:
Do we not want backports reviewed by someone? Or is it deemed sufficient that a maintainer tags a PR for automatic backporting?

Reply:
as a safer side let me add a check for single review instead of 2 reviews